### DISCUSSION TITLE
[Question]: A consideration for the future: Plus Plan for custom label maybe?

### DISCUSSION THREAD

**Discussion by Maverick87Shaka** [blakeblackshear/frigate#13560] [Ask A Question]
### What is your question?

Frigate is really interesting project, and while is totally understandable current dataset of object since the nature of the project, I do believe that rather than offer Plus plan "just" for custom trainee to have less false positive and so on, It will be interesting to use it to "drive" new object definition.
I mean, in the FAQ there is a kind of future vision to add more labels and so on, but It would be interesting if Plus (or a new plan) can cover this feature, in this way it's possible to use this gem even for further business use case (while keeping the fact that you're not going to re-sell, share and stuff ), but it can become even an interesting internal tools to improve business efficiency/safety. 

It's just my consideration based on some working experience.

Cheers

**Comment by NickM-27** [comment]
This was discussed on the original post https://github.com/blakeblackshear/frigate/discussions/1043
